### PR TITLE
[ADD] logistic_requisition_multicurrency

### PR DESCRIPTION
**Pull request by yvaucher**
Extraction and improvement of multicurrency in a new module

### CODE REVIEW

**Comment by gurneyalex** [comment]
can you add an automated test?

File: logistic_requisition_multicurrency/README.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Logistics Requisition - Multicurrency
+=====================================
+
+This module adds a fields on each obeject Logistics Requisition, Logistics Requisition Line
```
Comment (gurneyalex) [inline logistic_requisition_multicurrency/README.rst]: s/fields/field/
s/obeject/object/

**Comment by gurneyalex** [comment]
once typos in README.rst are fixed, :+1:

**Comment by foutoucour** [comment]
:+1:

**Comment by yvaucher** [comment]
typos fixed

**Pull request merged**